src/useForm.tsx

```diff
@@ -25,11 +25,17 @@ export type UseForm<T> = {
   reset: () => void
 }
 
+export type Option<T> = {
+  onStateChange: (value: T) => void
+  delay: number
```

Comment:
@teeeteee  -- I'm curious why we need a delay parameter in the first place? What problem does setting that to non-0 solve?

Reply:
`onStateChange` is wrapped in a debounce function and the delay parameter is passed in as param to the dobounce func